modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/legacy_appeals_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'caseflow/service'
+require 'common/exceptions'
+
+class AppealsApi::V1::DecisionReviews::LegacyAppealsController < AppealsApi::ApplicationController
+  SSN_REGEX = /^[0-9]{9}$/.freeze
+
+  skip_before_action(:authenticate)
+  before_action :validate_headers, only: %i[index]
+
+  EXPECTED_HEADERS = %w[X-VA-SSN X-VA-File-Number].freeze
+
+  def index
+    get_legacy_appeals_from_caseflow
+
+    render_response(caseflow_response)
+  end
+
+  private
+
+  attr_reader :caseflow_response
+
+  def get_legacy_appeals_from_caseflow
+    @caseflow_response = Caseflow::Service.new.get_legacy_appeals headers: headers
```

Comment:
We should probably add a timeout to this request, right? We can't be sure that the network request will always succeed